helm/aad-pod-identity-chart/templates/identities.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: "aadpodidentity.k8s.io/v1"
+kind: AzureIdentity
+metadata:
+ name: {{ .Values.azureIdentity.name }}
+spec:
+ type: {{ .Values.azureIdentity.type }}
+ ResourceID: {{ .Values.azureIdentity.resourceID }}
+ ClientID: {{ .Values.azureIdentity.clientID }}
```

Comment:
let's make `ResourceID` and `ClientID` `required`
https://github.com/virtual-kubelet/virtual-kubelet/blob/master/charts/virtual-kubelet/templates/deployment.yaml#L65